modules/virt-downloading-virtctl-client-online.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
```

Comment:
Are we using this file somewhere? Should not we delete it?

Reply:
Should've been deleted this pull.